srcpkgs/git-delete-merged-branches/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'git-delete-merged-branches'
+pkgname=git-delete-merged-branches
+version=7.4.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-wheel"
+depends="python3-colorama python3-prompt_toolkit"
+checkdepends="${depends} python3-parameterized python3-pytest"
+short_desc="CLI tool to delete merged git branches"
+maintainer="icp <pangolin@vivaldi.net>"
+license="GPL-3.0-or-later"
+homepage="https://pypi.org/project/git-delete-merged-branches/"
```

Comment:
```suggestion
homepage="https://github.com/hartwork/git-delete-merged-branches"
changelog="https://github.com/hartwork/git-delete-merged-branches/releases"
```

Reply:
IMO the pypi page in this case is a better homepage since it has more information readily available about the project. Even the Github `About` section links to the pypi page.

Reply:
The metadata in `setup.py` and the manpage point to github, as does the "homepage" link in pypi.

Either way is ok with me, but whatever your choice can you please add the changelog line? I find it quite useful (I have a script to grab changelogs and it works good with github releases page https://gist.github.com/tornaria/38b82815f5705077244ef4bb0f685e12).